### ISSUE TITLE
Fix/update the documentation for CALWF3 and associated standalone functions

### ISSUE THREAD

**Issue by mdlpstsci** [spacetelescope/wfc3tools#67]
The current on-line documentation for calwf3, as well as its components which can be run in standalone mode, is incorrect and out-of-date. The documentation and any associated diagrams need to be updated to reflect the true state of the pipeline software. This issue is related to Git PR#61. This issue is scheduled as JIRA HLA-759.

**Comment by mackjenn** [comment]
Currently, [readthedocs](https://wfc3tools.readthedocs.io/en/latest/wfc3tools/calwf3.html#uvis-pipleline) lists UVIS steps in a bulleted list (below Fig 3), with 'Unit conversion' coming just before FLASHCORR, which is incorrect.

Below that is a section called 'Unit Conversion to Electrons', which states:
The UVIS image is multiplied by gain right after BIASCORR, converting it to ELECTRONS. This step is no longer embedded within FLATCORR.

This is also incorrect, since it's applied after FLATCORR.  Your testing notes on this were in the discussion on the Saturation Maps [issue 558 ](https://github.com/spacetelescope/hstcal/issues/558)

We also discovered that the reference files FLSHFILE, SINKFILE, DARKFILE are in units of either electrons or electrons/sec, and that calwf3 applies the gain to these files before correcting the data which is still in units of DN.

I'm tagging @starivera on this ticket for followup.  Do we need to open a new ticket separate from the SATUFILE discussion to update readthedocs?

**Comment by mdlpstsci** [comment]
@mackjenn No. We can use this ticket and associated PR #68.  Thank you for the reminder on these issues.  I will schedule the doc update for the next sprint.

**Comment by pllim** [comment]
HLA-759 is marked as done, so closing.

**Comment by pllim** [comment]
If anything, we need to reduce duplication with WFC3 DHB, remove text from the doc in this repo and defer as much as we can to the DHB. So if you see anything else, please open new issue. Thanks!
